typescript/sdk/src/metadata/agentConfig.ts

```diff
@@ -63,6 +64,13 @@ const AgentSignerAwsKeySchema = z
   .describe(
     'An AWS signer. Note that AWS credentials must be inserted into the env separately.',
   );
+const AgentSignerCosmosKeySchema = z
```

Comment:
I think @jmrossy has some kind of logic that we can enforce to only show up/require when the protocol type is cosmos

Reply:
added a `refine`